engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/AliasAnalysis.scala

```diff
@@ -97,7 +97,7 @@ case object AliasAnalysis extends IRPass {
     ir: Module,
     moduleContext: ModuleContext
   ): Module = {
-    ir.copyWithBindings(bindings = ir.bindings.map(analyseModuleDefinition))
+    ir.copyWithBindings(ir.bindings.map(analyseModuleDefinition))
```

Comment:
In these refactorings, I am removing named arguments when calling Java from Scala, because:
- It sometimes causes compilation errors with weird messages.
- IntelliJ always treats it as an error.
- Despite the fact that `runtime-parser` is compiled with `-parameters` flag.